src/rfm.rs

```diff
@@ -6,8 +6,9 @@ use embedded_hal::digital::v2::OutputPin;
 use crate::cs::{CsGuard, NoCs};
 use crate::error::{Error, Result};
 use crate::registers::{
-    ContinuousDagc, DioMapping, DioPin, FifoMode, LnaConfig, Mode, Modulation, Pa13dBm1, Pa13dBm2,
-    PacketConfig, PacketFormat, Registers, RxBw, RxBwFreq, SensitivityBoost,
+    ContinuousDagc, DioMapping, DioPin, FifoMode,  IrqFlags1, IrqFlags2, LnaConfig, Mode,
```

Comment:
Ah one space slipped through, could you please fix that and squash the changes to a single commit?